src/components/Guides.tsx

```diff
@@ -0,0 +1,253 @@
+import React from 'react';
+import { Flex, Alert, Txt } from 'rendition';
+import { DownloadImage } from './DownloadImage';
+import { ExternalLink } from './ExternalLink';
+import insertCard from '../img/insert-sd.gif';
+import flashCard from '../img/etcher.gif';
+import tasksImg from '../img/tasks.png';
+import { DownloadEtcher } from './DownloadEtcher';
+import { LazyImage } from './LazyImage';
+import BalenaSdk from 'balena-sdk';
+
+const renderGenericInfo = (deviceName?: string) => (
+	<Txt.p>
+		Getting started on a <Txt.span bold>{deviceName}</Txt.span> is simple!
+		Follow these steps to download our ready-made operating system, flash it to
+		an SD Card, and begin crunching data to help scientists!
```

Comment:
This is not always an sd card, so maybe we want to say either `SD card or storage drive`, but others might have a better suggestion :)

Reply:
@sradevski Yes, this was done to enable that. We'll need a list of devices without SD card to do that.